src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age
+      && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+
+    {person.isMarried
+      && person.sex === 'f'
+      && (
+        <p className="Person__partner">
+          {`${person.partnerName} is my husband`}
+        </p>
+      )
+    }
+
+    {person.isMarried
+      && person.sex === 'm'
+      && (
+        <p className="Person__partner">
+          {`${person.partnerName} is my wife`}
+        </p>
+      )
+    }
+
+    {!person.isMarried
+      && (
+        <p className="Person__partner">
+          I am not married
+        </p>
+      )
+    }
```

Comment:
You can do it in simple way.

```suggestion
     <p className="Person__partner">
        {person.isMarried ? `${personPartner}` : `I am not married`}
      </p>
```

And create `personPartner` variable inside your `Person` component.
This `personPartner` will return:

```
person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`
```